src/lib/utilities/route-for.ts

```diff
@@ -62,7 +63,7 @@ export const routeForWorkflow = ({
   run,
   ...parameters
 }: WorkflowParameters): string => {
-  const wid = encodeURIComponent(workflow);
+  const wid = encodeURIForSvelte(workflow);
```

Comment:
This is fine for now, but maybe later we come up with a better name. 😂

Reply:
let's see maybe they resolve this issue by the time of v1.0.0 :) https://github.com/temporalio/ui/pull/491/files#diff-f21ba85e44db5049b5d73da8f05e545676388faba448b457f8eb61ace4a5edfcR2

Reply:
let's see maybe they resolve this issue by the time of v1.0.0 :) https://github.com/temporalio/ui/pull/491/files#diff-f21ba85e44db5049b5d73da8f05e545676388faba448b457f8eb61ace4a5edfcR2